LICENSE

```diff
@@ -1,5 +1,6 @@
-		    GNU GENERAL PUBLIC LICENSE
-		       Version 2, June 1991
+Valid-License-Identifier: GPL-2.0+
+
+       GNU General Public License v2.0 or later
 
```

Comment:
Hi @hwspeedy  this PR already covers the issue of uber-graph #702 ? as i can see, GTK3 does use it! based on CMAkelist file depends

**NOTE** what happened with the "forum like" place? where we discuss things?

Reply:
@mckaygerhard - I am trying to keep the community together.
- so my repo was scratched to make this commit as small as possible. This PR is all that is needed to move forward - GTK3 with uber-graph with correct licensing and a maintainer change, so that we all can help on this project.

As soon as @lpereira has transferred the repo, I will recreate discussion for entire project.

I have around 20 commits lined up for you and others to provide feedback on. And we have a discussion about dependencies for packaging. It will be fun - we will get hardinfo released.